### PR TITLE
Fix yq issues

### PR DESCRIPTION
**Pull request by teawater**
1. Set version to 2.3.0
2. Check the fail of curl

### CODE REVIEW

**Comment by lifupan** [comment]
The CI failed on fedora is the same with the failed on https://github.com/kata-containers/agent/pull/490, I'm not sure is this failed related with that PR, I'll have a look on this failed later.

File: .ci/install-yq.sh [hunk 1]
```diff
@@ -56,12 +56,7 @@ function install_yq() {
 		die "Please install curl"
 	fi
 
-	# Workaround to get latest release from github (to not use github token).
-	# Get the redirection to latest release on github.
-	yq_latest_url=$(curl -Ls -o /dev/null -w %{url_effective} "https://${yq_pkg}/releases/latest")
-	# The redirected url should include the latest release version
-	# https://github.com/mikefarah/yq/releases/tag/<VERSION-HERE>
-	yq_version=$(basename "${yq_latest_url}")
+	local yq_version=2.3.0
```
Comment (egernst) [inline .ci/install-yq.sh]: @grahamwhaley - can you put your @jodh-intel hat on and PTAL?

Comment (grahamwhaley) [reply .ci/install-yq.sh]: I think this is OK, to pin, to get stability. What I don't quite follow is that the curl of github release `latest` should be picking up the latest released version (which is 2.3.0 for yq right now), and not any `HEAD` version.
Thus, I actually think:
 - this change is currently benign
 - but in the long term, pinning to a known version probably gets us more stability than not pinning - but at the cost that we have to track.
Hmm, actually then, @chavafg - do we think this `yq` version number should be wired into the `versions.yaml` files, and not hard wired here in a script??

Comment (teawater) [reply .ci/install-yq.sh]: @egernst @grahamwhaley I agree with put the request version to versions.yaml.
But yq is used to parse versions.yaml.

Comment (grahamwhaley) [reply .ci/install-yq.sh]: @teawater - heh, hah!
In which case, I have no issue with this - as long as we remember it is hard wired ;-)

**Comment by advancedwebdeveloper** [comment]
I got an issue on Ubuntu 20 x86_64 - I am using [gollvm](https://go.googlesource.com/gollvm/) : 

> ~/kata-containers/src/runtime$ make GOPATH=~/go
> /bin/sh: 1: test: Illegal number: 15rc2
> golang.mk:60: *** "ERROR: golang minor version too old: got 1.15rc2, need atleast 1.11".  Stop.

In my case

> $ yq -V
> yq version 3.4.0
> 
> ~/kata-containers/src/runtime$ go env && go version
> GO111MODULE=""
> GOARCH="amd64"
> GOBIN=""
> GOCACHE="/home/oceanfish81/.cache/go-build"
> GOENV="/home/oceanfish81/.config/go/env"
> GOEXE=""
> GOFLAGS=""
> GOHOSTARCH="amd64"
> GOHOSTOS="linux"
> GOINSECURE=""
> GOMODCACHE="/home/oceanfish81/go/pkg/mod"
> GONOPROXY=""
> GONOSUMDB=""
> GOOS="linux"
> GOPATH="/home/oceanfish81/go"
> GOPRIVATE=""
> GOPROXY="https://proxy.golang.org,direct"
> GOROOT="/usr/local"
> GOSUMDB="sum.golang.org"
> GOTMPDIR=""
> GOTOOLDIR="/usr/local/tools"
> GCCGO="/usr/local/bin/llvm-goc"
> AR="ar"
> CC="/usr/bin/clang"
> CXX="/usr/bin/clang++"
> CGO_ENABLED="1"
> GOMOD="/home/oceanfish81/kata-containers/src/runtime/go.mod"
> CGO_CFLAGS="-g -O2"
> CGO_CPPFLAGS=""
> CGO_CXXFLAGS="-g -O2"
> CGO_FFLAGS="-g -O2"
> CGO_LDFLAGS="-g -O2"
> PKG_CONFIG="pkg-config"
> GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build179553929=/tmp/go-build -gno-record-gcc-switches -funwind-tables"
> go version go1.15rc2 gollvm LLVM 12.0.0git linux/amd64

Ivan

**Pull request merged**